beacon_node/beacon_chain/src/data_column_verification.rs

```diff
@@ -233,11 +234,17 @@ impl<T: BeaconChainTypes, O: ObservationStrategy> GossipVerifiedDataColumn<T, O>
 #[ssz(struct_behaviour = "transparent")]
 pub struct KzgVerifiedDataColumn<E: EthSpec> {
     data: Arc<DataColumnSidecar<E>>,
+    #[ssz(skip_serializing, skip_deserializing)]
```

Comment:
Do we still need `Encode` and `Decode` here? might be able to remove it now we don't store overflow cache anymore.